proxy/hdrs/HTTP.h

```diff
@@ -796,12 +796,13 @@ HTTPHdr::create(HTTPType polarity, HdrHeap *heap)
 }
 
 inline void
-HTTPHdr::clear()
+HTTPHdr::destroy()
```

Comment:
I'm seeing several other usages of `HTTPHdr::clear()` such as:

`Http2UpgradeContext::~Http2UpgradeContext()`
`HttpTransactCache::calculate_quality_of_match()`

And `HTTPHdr::unmarshal()` 

Am I missing something?